packages/app-builder-lib/scheme.json

```diff
@@ -802,6 +802,11 @@
                 "window": {
                     "$ref": "#/definitions/DmgWindow",
                     "description": "The DMG windows position and size."
+                },
+                "sign": {
+                    "default": true,
```

Comment:
According to https://github.com/electron-userland/electron-builder/commit/ef0028d#diff-95f823016d05ece1f904f6c59a352ebaR190 the default is `false`. That file might need to be updated